### ISSUE TITLE
Add button "Save as image"

### ISSUE THREAD

**Issue by iliyan-velichkov** [eclipse/dirigible#3270]
Add a button like "Save as image" which exports the model to image.
![image](https://github.com/eclipse/dirigible/assets/5058839/bf601a87-18fd-4785-a6e8-ae0844acde71)

**Comment by ThuF** [comment]
The "Show" button can be used to display the diagram in full screen and the "Print Page" functionality from the browser can be used to save a PDF version of the diagram.

**Comment by iliyan-velichkov** [comment]
> The "Show" button can be used to display the diagram in full screen and the "Print Page" functionality from the browser can be used to save a PDF version of the diagram.

Yes, but I don't want PDF, I want an image. Most tools have this functionality.
